src/test/integration/IntegrationDeployer.t.sol

```diff
@@ -131,503 +94,445 @@ abstract contract IntegrationDeployer is ExistingDeploymentParser {
     }
 
-    /// Deploy EigenLayer locally
-    function _setUpLocal() public virtual noTracing {
-        console.log("Setting up `%s` integration tests:", "LOCAL".yellow().bold());
+    /// @dev Configures the possible asset types for the random users.
+    function _configAssetTypes(uint _assetTypes) internal virtual {
+        assertTrue((assetTypes = _assetTypes) != 0, "_configAssetTypes: no asset types selected");
+    }
 
-        // Deploy ProxyAdmin
-        eigenLayerProxyAdmin = new ProxyAdmin();
-        executorMultisig = address(eigenLayerProxyAdmin.owner());
+    /// @dev Configures the possible user types for the random users.
+    function _configUserTypes(uint _userTypes) internal virtual {
+        assertTrue((userTypes = _userTypes) != 0, "_configUserTypes: no user types selected");
+    }
 
-        // Deploy PauserRegistry
-        address[] memory pausers = new address[](1);
-        pausers[0] = pauser;
-        eigenLayerPauserReg = new PauserRegistry(pausers, unpauser);
+    /// @dev Configures the maximum number of unique assets a user can hold.
+    function _configAssetAmounts(uint _maxUniqueAssetsHeld) internal virtual {
+        if (_maxUniqueAssetsHeld > allStrats.length) _maxUniqueAssetsHeld = allStrats.length;
+        assertTrue((maxUniqueAssetsHeld = _maxUniqueAssetsHeld) != 0, "_configAssetAmounts: invalid 0");
+    }
 
-        // Deploy mocks
-        emptyContract = new EmptyContract();
+    function FOUNDRY_PROFILE() internal view returns (string memory) {
+        return cheats.envOr(string("FOUNDRY_PROFILE"), string("default"));
+    }
 
-        // Matching parameters to testnet
-        DELEGATION_MANAGER_MIN_WITHDRAWAL_DELAY_BLOCKS = 50;
-        DEALLOCATION_DELAY = 50;
-        ALLOCATION_CONFIGURATION_DELAY = 75;
+    function eq(string memory a, string memory b) internal pure returns (bool) {
+        return keccak256(bytes(a)) == keccak256(bytes(b));
+    }
+
+    /// -----------------------------------------------------------------------
+    /// Environment Setup
+    /// -----------------------------------------------------------------------
 
-        REWARDS_COORDINATOR_CALCULATION_INTERVAL_SECONDS = 86_400;
-        REWARDS_COORDINATOR_MAX_REWARDS_DURATION = 6_048_000;
-        REWARDS_COORDINATOR_MAX_RETROACTIVE_LENGTH = 7_776_000;
-        REWARDS_COORDINATOR_MAX_FUTURE_LENGTH = 2_592_000;
-        REWARDS_COORDINATOR_GENESIS_REWARDS_TIMESTAMP = 1_710_979_200;
+    /// @dev Sets up the integration tests for local.
+    function _setUpLocal() public virtual {
+        console.log("Setting up `%s` integration tests:", "LOCAL".yellow().bold());
+
+        // Deploy ProxyAdmin, PauserRegistry, and executorMultisig.
+        config.governance.proxyAdmin = new ProxyAdmin();
+        config.governance.pauserRegistry = new PauserRegistry(PAUSER.toArray(), UNPAUSER);
+        config.governance.executorMultisig = address(proxyAdmin().owner());
 
         _deployProxies();
```

Comment:
I see we got rid of `_deployImplementations` here, which is fine - I think moving this to `_upgradeProxies` is clean.

That said, it feels little inconsistent that the `Config` contract has fields for "impls" - but I don't think we set them anywhere. IIUC, these are only set if we're doing a fork test and they get read from config?

We should either:
* Remove impls from config/parsing entirely. Maybe they're just not needed!
* Make sure config is updated with impls no matter what env you're running in